app/res/layout/screen_login.xml

```diff
@@ -86,21 +86,19 @@
                                 android:textSize="@dimen/text_medium"
                                 android:visibility="gone"/>
 
-                            <EditText
+                            <AutoCompleteTextView

```

Comment:
As far as I could find in my searching, Android doesn't offer an out-of-the-box combobox. I'm sure I could make it custom in some way, but it would have taken a lot longer and it didn't seem worth the time right now. Especially given how tangential this is to the actual feature.